spec/fixtures/lod_loc_term_found.rdf.xml

```diff
@@ -116,7 +116,7 @@
       </madsrdf:Topic>
     </madsrdf:hasVariant>
     <madsrdf:editorialNote>headings beginning with the word [Scientific;] and subdivision [Science] under ethnic groups and individual wars, e.g. [World War, 1939-1945--Science]</madsrdf:editorialNote>
-    <identifiers:lccn xmlns:identifiers="http://id.loc.gov/vocabulary/identifiers/">sh 85118553</identifiers:lccn>
+    <identifiers:lccn xmlns:identifiers="http://id.loc.gov/vocabulary/identifiers/">sh85118553</identifiers:lccn>
```

Comment:
This is an ugly change.  I am in discussions with loc.  For the id predicate, the id includes a blank (e.g. `sh 85118553`), but the uri does not.  The current loc config expects the id to be passed in without a blank so it can be substituted into the URL template.  But the code that tries to identify the URI from the results, looks for the id as the object value and assumes that the URI for the matching triple is the subject_uri being fetched.  In this case, the id in the object value does want the blank.

For now, I am removing the blank from the fixture data file representing loc data.  I believe the long term fix is to force loc config to require the URI to be passed in for find.  Then we don't have to figure out how to translate from id to uri.

My hesitation to go there now is that will cause a major version as it will not be backward compatible for existing loc configs.

The other option is to add kludge code for loc that adds/removes the blank as needed for loc only.  Yuck.